Index: src/main/java/dtu/system/Activity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dtu.system;\n\npublic class Activity {\n    String name;\n\n\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/dtu/system/Activity.java b/src/main/java/dtu/system/Activity.java
--- a/src/main/java/dtu/system/Activity.java	(revision 84d0a41f9fdf1c6c21bafba4482c0e452f20efc9)
+++ b/src/main/java/dtu/system/Activity.java	(date 1648150823314)
@@ -1,7 +1,14 @@
 package dtu.system;
 
+import java.util.ArrayList;
+import java.util.Calendar;
+
 public class Activity {
     String name;
+    Calendar startDate, endDate;
+    double timeBudget;
+    int weekNumber;
+    ArrayList<DevelopmentEmployee> workingDevelopers;
 
 
 
